tests/pytest_tests/system_tests/conftest.py

```diff
@@ -1017,3 +1017,15 @@ def match(self, request):
         )
 
     yield helper
+
+
+@pytest.fixture(scope="function")
+def debug_logs():
```

Comment:
we should avoid non-automatic fixtures that don't return a value because their usage isn't clear in tests:

```python
# debug_logs will be grayed out like an unused variable in most IDEs.
def test_thing(..., debug_logs):
    # From the parameter name, it's not clear what effect it has on the test!
    # "enable_debug_logs" would be better, but the fact that it's grayed out
    # still makes it seem unused.
    ...
```

In this case, monkeypatching the env explicitly in tests seems like a better approach. I would use `pytest`'s built-in `monkeypatch` fixture---that saves you from having to nest everything inside a `with` block:

```python
def test_thing(..., monkeypatch):
    monkeypatch.setenv("WANDB_DEBUG", "true")
    ...
```